src/dataset/masked.py

```diff
@@ -46,8 +46,28 @@ def load_dataset(dataset: str) -> Tuple[Iterator, Iterator]:
     return train_iter, test_iter
 
 
-def make_metadata(dataset: str
-                  ) -> Tuple[MaskedDatasetMetadata, MaskedDatasetMetadata]:
+def load_replace_data(dataset: str) -> Tuple[Iterator, MaskedDatasetMetadata]:
+    replace_gen = ImageDataGenerator(rescale=10)
```

Comment:
Removing rescale at some point?

Reply:
Just for the books, I did, then we decided to keep it.